examples/Logging.ipynb

```diff
@@ -23,147 +22,149 @@
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "source": [],
    "outputs": [],
-   "source": []
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
    "source": [
     "data_file = \"data/lending_club_1000.csv\""
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "scrolled": true
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
    "source": [
     "data = pd.read_csv(os.path.join(data_file))\n",
     "data"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "scrolled": true
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "We should see a Pandas dataframe containing the 1000 rows of our Lending Club data sample.\n",
     "\n",
     "Now that we have the raw data, we can pass it into the whylogs logger. It is often useful to pass a string label such as \"demo.data\" along with the dataset for future reference.\n",
     "\n",
     "The `log_dataframe` function will profile the given dataset using the whylogs library. When we capture the logger response, we can interact with the generated profiles."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "profile = session.log_dataframe(data, 'test.data')\n",
     "print(profile)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "The flat summary, histograms, and frequency information can be found inside this summary object. \n",
     "\n",
     "For more information about the contents of these objects, consult the `Analysis.ipynb` notebook."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "summary = profile.flat_summary()\n",
     "flat_summary = summary['summary']\n",
     "flat_summary"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "print(flat_summary[\"column\"].unique())"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "histograms = summary['hist']\n",
     "histograms[\"delinq_amnt\"]"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "frequencies = summary['frequent_strings']\n",
-    "frequencies['num_sats']"
-   ]
+    "frequent_strings = [key for key in frequencies.keys()]\n",
+    "print(frequent_strings)\n",
```

Comment:
This should avoid errors reading a specific frequent string from the lending club data.